account_invoice_validate_send_email/readme/CONTRIBUTORS.rst

```diff
@@ -1 +1,3 @@
 * JackLe <lehoangan1988@gmail.com>
+* Yoshi Tashiro <tashiro@quartile.co>
+* Tomohiro Abe <tomohiro@quartile.co>
```

Comment:
Don't bother updating the contributors for non-OCA modules.

Reply:
Understand.
Now re-generated readme and description files.